src/main/java/org/example/Main.java

```diff
@@ -2,6 +2,17 @@
 
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+
+        for (int i = 0; i <= 100; i++) {
+            if (i % 3 == 0 && i % 5 == 0) {
```

Comment:
Jakbyś tak spróbował wydzielić to co masz w tej pętli for do jakiejś oddzielnej metody.